client/checkout/api/index.js

```diff
@@ -362,56 +345,30 @@ export default class WCPayAPI {
 	 * @param {Object} productData Product data.
 	 * @return {Promise} Promise for the request to the server.
 	 */
-	paymentRequestAddToCart( productData ) {
-		return this.request( getPaymentRequestAjaxURL( 'add_to_cart' ), {
-			security: getPaymentRequestData( 'nonce' )?.add_to_cart,
+	expressCheckoutECEAddToCart( productData ) {
+		return this.request( getExpressCheckoutAjaxURL( 'add_to_cart' ), {
```

Comment:
@frosso Apologies if this is an obvious question, but why we avoid `ece` prefix for this use-case while keeping it for others `ece_get_cart_details` `ece_get_selected_product_data` `ece_create_order` etc.?

Reply:
Sorry I missed this initially - connected to the above comment, I noticed that the same `add_to_cart` endpoint (and handler) is used for both the ECE and the WooPay implementations. Their JS implementation changes slightly because they use slightly different ways of passing nonces and building the endpoint URL.